Workaround for `ImportError: cannot import name 'DEV_PKGS' from 'pip._internal.commands.freeze'`

jsvob opened the pull request:
Workaround for `ImportError: cannot import name 'DEV_PKGS' from 'pip._internal.commands.freeze'` when setting up a local dev environment or when applying requirements.txt.

jobselko commented:
I think we should update `pip-tools` instead of downgrading `pip`. See https://github.com/jazzband/pip-tools/releases/tag/7.0.0

jsvob commented:
Updated the branch https://github.com/RedHatProductSecurity/osidb/commits/pin_pip_23_1 with Jitka's suggestion and a makefile fix, but the pull request still shows the outdated now-removed commit. IDK why.
